marlowe-playground-client/src/Reachability.purs

```diff
@@ -268,10 +272,21 @@ stepAnalysis settings isReachable rad =
   in
     if thereAreMore then do
       assign _analysisState (ReachabilityAnalysis (InProgress newRad))
+      refreshEditor
       response <- checkContractForReachability settings (newRad.currContract) (newRad.originalState)
       updateWithResponse settings (InProgress newRad) response
-    else
-      pure $ finishAnalysis newRad
+    else do
+      let
+        result = finishAnalysis newRad
+      assign _analysisState (ReachabilityAnalysis result)
+      refreshEditor
+      pure result
+  where
+  refreshEditor = do
+    mContent <- query _marloweEditorSlot unit (Monaco.GetText identity)
+    case mContent of
+      Just content -> void $ query _marloweEditorSlot unit $ Monaco.SetText content unit
+      Nothing -> pure unit
```

Comment:
presumably this is to trigger the linter? It's rather unsatisfactory 😞  I wonder if it has performance implications with larger contracts? I wonder if it's possible to trigger the linter directly? Also if there is no other way, when you see
```purescript
case x of
  Nothing -> pure unit
  Just y -> void $ ...
```
You are probably better using `for_ x \y -> ...`

Reply:
Yes, I don't like it much either, but I am not sure how to go about it.
I am modifying it so that it only calls it when there are new counter examples, so it shouldn't be called very often...